std_msgs/Header robot_map_frame
---
geometry_msgs/PointStamped[] frontiers